README.md

```diff
@@ -26,7 +26,7 @@ golint prints out style mistakes.
 
 Golint differs from govet. Govet is concerned with correctness, whereas
 golint is concerned with coding style. Golint is in use at Google, and it
-seeks to match the accepted style of the open source Go project.
+seeks to match the accepted style of the open source Go project, [as described here](https://github.com/golang/go/wiki/CodeReviewComments).
```

Comment:
That's not really a true statement either. I'd phrase things like:

Golint makes suggestions for many of the mechanically checkable items listed in
Effective Go and the CodeReviewComments wiki page.